### PR TITLE
ng-model fix to category

### PR DESCRIPTION
**Pull request by radhikabhanu**

### CODE REVIEW

File: static/templates/project/categories-create.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div layout="row">
 	<md-subheader class="md-no-sticky">At D&aelig;mo we want to make task creation simple and intuitive. Get started now by choosing from one of our predesigned templates or by customizing your own.</md-subheader>
 </div>
-<md-radio-group ng-model="project.templateName">
+<md-radio-group ng-model="project.currentProject.templateName">
```
Comment (dmorina) [inline static/templates/project/categories-create.html]: this seems to use (and have used) the wrong field in the currentProject

**Comment by nistala** [comment]
@radhikabhanu could you please add a description to the PR

**Comment by dmorina** [comment]
#453 will fix this

**Comment by mbernst** [comment]
Process question: is there any way @radhikabhanu
<https://github.com/radhikabhanu> could or should have known about the
impending fix so the PR effort wasn't wasted? Should they have checked with
#researchengineering somehow?

**Comment by dmorina** [comment]
there was a discussion #research-engineering channel about this PR
no updates have been pushed anymore

**Comment by shirishgoyal** [comment]
This PR made a very minor change but #453 resolves it fully in addition to persistence issues.

**Comment by mbernst** [comment]
OK, thanks!

**Pull request closed without merging**